stdlib/public/core/Mirror.swift

```diff
@@ -481,19 +481,19 @@ public struct Mirror {
     children: C,
     displayStyle: DisplayStyle? = nil,
     ancestorRepresentation: AncestorRepresentation = .generated
-  ) where C.Element == Child { Builtin.unreachable() }
+  ) where C.Element == Child { self.init(reflecting: subject) }
```

Comment:
This is a workaround for a compiler crash in `SILInstruction::verifyDebugInfo()` that I can reproduce on a small reproducer:

```
// RUN: %target-swift-frontend -Osize -parse-stdlib -enable-ossa-modules -emit-sil %s -o %t.sil

import Swift

public struct A {
  public init(_ pairs: KeyValuePairs<Any, Any>) {
    Builtin.unreachable()
  }
}
```

I filed this bug for it, https://bugs.swift.org/browse/SR-15300, but here I think we can just apply this simple workaround because these are all unused stubs anyway.

Reply:
I would feel more comfortable with a comment referring back to SR-15300.  Would you mind just throwing that test case into the tests directory with an XFAIL?  I know it is in the SR, but would be nice to catch it if it accidentally gets resolved.